doc/METADATA.ini

```diff
@@ -1211,6 +1211,13 @@ description= check if value is a valid rgbcolor in hexformat
 # Old stuff inherited from filesys plugin
 #
 
+[check/memoryvalue]
+type= empty
+status = implemented
+usedby/plugin= memoryvalue
+description= check if value is a valid memoryvalue
```

Comment:
If you do not know what a memoryvalue is, this description does not help. An example would be very helpful.